docs/guidance/tls.md

```diff
@@ -154,8 +154,71 @@ echo $? # 0
 ray health-check --address $RAY_IP:6379
 
 # If you add `DNS.3 = $RAY_IP` to the [alt_names] section in `gencert_head.sh`,
-# the head Pod will generate the certificate of $RAY_IP. 
-# 
+# the head Pod will generate the certificate of $RAY_IP.
+#
 # For KubeRay versions prior to 0.5.0, this step is necessary because Ray workers in earlier
 # versions use $RAY_IP to connect with Ray head.
 ```
+
+# Step 5: Connect to the cluster with Ray client using TLS for interactive development
+To learn more, please check [interactive development](https://docs.ray.io/en/latest/cluster/running-applications/job-submission/ray-client.html#ray-client-interactive-development) and [TLS authentication](https://docs.ray.io/en/latest/ray-core/configure.html?highlight=tls#tls-authentication) for more detail.
+
+For instructions on connecting the Ray cluster from a Pod:
+```
+# Create a client pod and connect to cluster
+kubectl apply -f ray-operator/config/samples/ray-pod.tls.yaml
+kubectl logs ray-client-tls
+```
+Verify the output similar to:
+```
+{'CPU': 2.0, 'node:10.254.20.20': 1.0, 'object_store_memory': 771128524.0, 'memory': 3000000000.0, 'node:10.254.16.25': 1.0}
+```
+
+For instruction on connecting the Ray cluster from a Jupyter Notebook from your local machine, follow the instruction below:
```

Comment:
Would you mind removing the following section? There are some limitations in Ray client. By the way, the Ray community wants to de-emphasize Ray client and promote Ray job submission instead.

Reply:
Removed.